src/components/charts/AggregateLineChart.tsx

```diff
@@ -60,7 +60,9 @@ export const AggregateLineChart: React.FC<AggregateLineProps> = ({
             label: 'Before shutdown',
             fill: false,
             tension: 0.1,
-            pointBackgroundColor: CHART_COLORS.GREY,
+            borderColor: darkMode ? '#94a3b8' : '#c8cbcf',
+            backgroundColor: darkMode ? '#94a3b8' : '#c8cbcf',
+            pointBackgroundColor: darkMode ? 'white' : CHART_COLORS.GREY,
```

Comment:
Ideally we should add all these colors to the enum CHART_COLORS

Reply:
yeah some of them are just default chartjs colors 🤷